client/knode/validator/validator.go

```diff
@@ -113,16 +120,24 @@ func New(backend Backend, consensus *consensus.Consensus, config *params.ChainCo
 		signer:    types.NewAndromedaSigner(config.ChainID),
 		vmConfig:  vmConfig,
 		canStart:  0,
+		logger:    log.New("package", "knode/validator"),
 	}
 
+	validator.logger.SetHandler(log.MultiHandler(
+		log.StdoutHandler,
+		fileHandler,
+		fileHandlerErr,
+		fileHandlerCrit),
+	)
+
 	go validator.sync()
 
 	return validator
 }
 
 func (val *validator) sync() {
 	if err := SyncWaiter(val.eventMux); err != nil {
-		log.Warn("Failed to sync with network", "err", err)
+		val.logger.Crit("Failed to sync with network", "err", err)
```

Comment:
why is this now Critical?

Reply:
Because we can't start the validation without syncing

Reply:
that doesn't mean we should kill the client, can we return nil or not starting vlidation process

Reply:
The current flow is a one shot type of thing and we will need to modify the flow to take that into account. I will open a separate issue for that problem.